.github/workflows/maven-build.yml

```diff
@@ -121,7 +122,7 @@ jobs:
             /home/runner/work
           key: ${{ needs.build.outputs.cache_key }}
       - name: 📦 Publish to Maven Central
-        run: mvn --batch-mode deploy -P gpg-sign -P nexus-staging
+        run: mvn --batch-mode -Dmaven.test.skip=true deploy -P gpg-sign -P nexus-staging
```

Comment:
`-Dmaven.test.skip=true`this should not be here: we are publishing test jar as well